flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -32,6 +33,17 @@ import scala.collection.mutable
 
 class SqlITCase extends StreamingMultipleProgramsTestBase {
```

Comment:
In addition to an integration test we should also add tests based on `TableTestBase` which check the correct translation into `DataSetRelNode`s. See `WindowAggregateTest` as an example.